### PR TITLE
IPv4 location markup batch 3

### PR DESCRIPTION
**Pull request by Yullia**
IPv4 location markup batch 3 (100-150)

### CODE REVIEW

**Review by babenek — CHANGES_REQUESTED** [review]
use Info for an IP address markup (TRUE - mandatory, FALSE - not necessary)

File: meta/894e3377.csv [hunk 1]
```diff
@@ -750,25 +750,25 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 132126,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,381:381,F,F,,,F,F,,,,,0.00,,F,F,F,Other
 132127,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,382:382,F,F,,,F,F,,,,,0.00,,F,F,F,Other
 132128,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,383:383,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132129,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,384:384,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132130,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,385:385,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132131,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,386:386,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+132129,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,384:384,F,F,28,37,F,F,,,,,0.00,,F,F,F,Other
+132130,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,385:385,F,F,30,39,F,F,,,,,0.00,,F,F,F,Other
+132131,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,386:386,F,F,34,43,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/894e3377.csv:755]:
```suggestion
132129,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,384:384,F,F,28,37,F,F,,,Info,,0.00,,F,F,F,Other
132130,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,385:385,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
132131,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,386:386,F,F,34,43,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/894e3377.csv [hunk 2]
```diff
@@ -750,25 +750,25 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+132138,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,402:402,F,F,23,32,F,F,,,,,0.00,,F,F,F,Other
+132139,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,403:403,F,F,30,39,F,F,,,,,0.00,,F,F,F,Other
+132140,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,404:404,F,F,32,41,F,F,,,,,0.00,,F,F,F,Other
+132141,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,409:409,F,F,27,36,F,F,,,,,0.00,,F,F,F,Other
+132142,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,435:435,F,F,17,28,F,F,,,,,0.00,,F,F,F,Other
+132143,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,436:436,F,F,15,26,F,F,,,,,0.00,,F,F,F,Other
+132144,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2566:2566,F,F,5,12,F,F,,,,,0.00,,F,F,F,Other
+132145,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2567:2567,F,F,5,12,F,F,,,,,0.00,,F,F,F,Other
+132146,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2568:2568,F,F,5,12,F,F,,,,,0.00,,F,F,F,Other
+132147,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2569:2569,F,F,5,12,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/894e3377.csv:771]:
```suggestion
132133,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,395:395,F,F,29,38,F,F,,,Info,,0.00,,F,F,F,Other
132134,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,396:396,F,F,24,33,F,F,,,Info,,0.00,,F,F,F,Other
132135,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,399:399,F,F,22,31,F,F,,,Info,,0.00,,F,F,F,Other
132136,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,400:400,F,F,37,46,F,F,,,Info,,0.00,,F,F,F,Other
132137,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,401:401,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
132138,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,402:402,F,F,23,32,F,F,,,Info,,0.00,,F,F,F,Other
132139,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,403:403,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
132140,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,404:404,F,F,32,41,F,F,,,Info,,0.00,,F,F,F,Other
132141,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,409:409,F,F,27,36,F,F,,,Info,,0.00,,F,F,F,Other
132142,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,435:435,F,F,17,28,F,F,,,Info,,0.00,,F,F,F,Other
132143,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,436:436,F,F,15,26,F,F,,,Info,,0.00,,F,F,F,Other
132144,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2566:2566,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132145,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2567:2567,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132146,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2568:2568,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132147,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2569:2569,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/894e3377.csv [hunk 3]
```diff
@@ -750,25 +750,25 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
```
Comment (Yullia) [inline meta/894e3377.csv:771]:
```suggestion
132133,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,395:395,F,F,29,38,F,F,,,Info,,0.00,,F,F,F,Other
132134,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,396:396,F,F,24,33,F,F,,,Info,,0.00,,F,F,F,Other
132135,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,399:399,F,F,22,31,F,F,,,Info,,0.00,,F,F,F,Other
132136,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,400:400,F,F,37,46,F,F,,,Info,,0.00,,F,F,F,Other
132137,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,401:401,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
132138,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,402:402,F,F,23,32,F,F,,,Info,,0.00,,F,F,F,Other
132139,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,403:403,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
132140,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,404:404,F,F,32,41,F,F,,,Info,,0.00,,F,F,F,Other
132141,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,409:409,F,F,27,36,F,F,,,Info,,0.00,,F,F,F,Other
132142,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,435:435,F,F,17,28,F,F,,,Info,,0.00,,F,F,F,Other
132143,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,436:436,F,F,15,26,F,F,,,Info,,0.00,,F,F,F,Other
132144,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2566:2566,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132145,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2567:2567,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132146,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2568:2568,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132147,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2569:2569,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/99f9a6ea.csv [hunk 4]
```diff
@@ -91,6 +91,6 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 102346,bb68cd53,GitHub,99f9a6ea,data/99f9a6ea/test/bb68cd53.rb,98:98,F,F,,,F,F,,,,,0.00,,F,F,F,Generic Secret
 102347,de5ab07c,GitHub,99f9a6ea,data/99f9a6ea/test/de5ab07c.rb,27:27,F,F,,,F,F,,,,,0.00,,F,F,F,Generic Secret
 102348,de5ab07c,GitHub,99f9a6ea,data/99f9a6ea/test/de5ab07c.rb,140:140,F,F,,,F,F,,,,,0.00,,F,F,F,Generic Secret
-132153,06572a96,GitHub,99f9a6ea,data/99f9a6ea/other/06572a96.md,283:283,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+132153,06572a96,GitHub,99f9a6ea,data/99f9a6ea/other/06572a96.md,283:283,F,F,0,7,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/99f9a6ea.csv:94]:
```suggestion
132153,06572a96,GitHub,99f9a6ea,data/99f9a6ea/other/06572a96.md,283:283,F,F,0,7,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/a0cd6261.csv [hunk 5]
```diff
@@ -16,9 +16,9 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 2088,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,719:719,F,F,,,F,F,,,,,0,0,F,F,F,Predefined Pattern
 3756,0fadd2c6,GitHub,a0cd6261,data/a0cd6261/other/0fadd2c6.md,184:184,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 3783,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,45:45,T,F,,,F,F,,,,,0,0,F,F,F,Other
-3784,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,42:42,T,F,,,F,F,,,,,0,0,F,F,F,Other
-3785,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,31:31,T,F,,,F,F,,,,,0,0,F,F,F,Other
-3786,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,28:28,T,F,,,F,F,,,,,0,0,F,F,F,Other
+3784,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,42:42,T,F,36,50,F,F,,,,,0,0,F,F,F,Other
+3785,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,31:31,T,F,36,50,F,F,,,,,0,0,F,F,F,Other
+3786,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,28:28,T,F,36,50,F,F,,,,,0,0,F,F,F,Other
```
Comment (Yullia) [inline meta/a0cd6261.csv:21]:
```suggestion
3784,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,42:42,T,F,36,50,F,F,,,Info,,0,0,F,F,F,Other
3785,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,31:31,T,F,36,50,F,F,,,Info,,0,0,F,F,F,Other
3786,1282093a,GitHub,a0cd6261,data/a0cd6261/src/1282093a.txt,28:28,T,F,36,50,F,F,,,Info,,0,0,F,F,F,Other
```

File: meta/a0cd6261.csv [hunk 6]
```diff
@@ -43,8 +43,8 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 5389,919b979a,GitHub,a0cd6261,data/a0cd6261/other/919b979a.md,79:79,F,F,,,F,F,,,,,0,0,F,F,F,Other
 5610,6a1897e7,GitHub,a0cd6261,data/a0cd6261/other/6a1897e7.md,139:139,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 5616,0aec68ea,GitHub,a0cd6261,data/a0cd6261/other/0aec68ea.md,51:51,F,F,,,F,F,,,,,0,0,F,F,F,Predefined Pattern
-6867,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,96:96,F,F,,,F,F,,,,,0,0,F,F,F,Password
-6868,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,94:94,F,F,,,F,F,,,,,0,0,F,F,F,Password
+6867,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,96:96,F,F,13,22,F,F,,,,,0,0,F,F,F,Password
+6868,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,94:94,F,F,13,22,F,F,,,,,0,0,F,F,F,Password
```
Comment (Yullia) [inline meta/a0cd6261.csv:47]:
```suggestion
6867,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,96:96,F,F,13,22,F,F,,,Info,,0,0,F,F,F,Password
6868,dbef751c,GitHub,a0cd6261,data/a0cd6261/other/dbef751c.md,94:94,F,F,13,22,F,F,,,Info,,0,0,F,F,F,Password
```

File: meta/a0cd6261.csv [hunk 7]
```diff
@@ -340,24 +340,24 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
 131851,92025475,GitHub,a0cd6261,data/a0cd6261/other/92025475.md,32:32,T,F,0,36,F,F,,,,,0.00,,F,F,F,Other
 131852,92025475,GitHub,a0cd6261,data/a0cd6261/other/92025475.md,33:33,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132169,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,86:86,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132170,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,88:88,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132171,7f6a3252,GitHub,a0cd6261,data/a0cd6261/other/7f6a3252.md,134:134,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132172,e955e6ae,GitHub,a0cd6261,data/a0cd6261/other/e955e6ae.md,1010:1010,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+132169,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,86:86,T,F,32,45,F,F,,,,,0.00,,F,F,F,Other
+132170,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,88:88,T,F,11,24,F,F,,,,,0.00,,F,F,F,Other
+132171,7f6a3252,GitHub,a0cd6261,data/a0cd6261/other/7f6a3252.md,134:134,T,F,31,38,F,F,,,,,0.00,,F,F,F,Other
+132172,e955e6ae,GitHub,a0cd6261,data/a0cd6261/other/e955e6ae.md,1010:1010,T,F,42,49,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/a0cd6261.csv:346]:
```suggestion
132169,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,86:86,T,F,32,45,F,F,,,Info,,0.00,,F,F,F,Other
132170,d73e7ecc,GitHub,a0cd6261,data/a0cd6261/other/d73e7ecc.md,88:88,T,F,11,24,F,F,,,Info,,0.00,,F,F,F,Other
132171,7f6a3252,GitHub,a0cd6261,data/a0cd6261/other/7f6a3252.md,134:134,T,F,31,38,F,F,,,Info,,0.00,,F,F,F,Other
132172,e955e6ae,GitHub,a0cd6261,data/a0cd6261/other/e955e6ae.md,1010:1010,T,F,42,49,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/a0cd6261.csv [hunk 8]
```diff
@@ -340,24 +340,24 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+132177,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,249:249,T,F,30,37,F,F,,,,,0.00,,F,F,F,Other
+132178,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,250:250,T,F,5,12,F,F,,,,,0.00,,F,F,F,Other
+132179,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,742:742,T,F,7,18,F,F,,,,,0.00,,F,F,F,Other
+132180,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,743:743,T,F,7,18,F,F,,,,,0.00,,F,F,F,Other
+132181,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,744:744,T,F,7,18,F,F,,,,,0.00,,F,F,F,Other
+132182,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,745:745,T,F,7,18,F,F,,,,,0.00,,F,F,F,Other
+132183,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,751:751,T,F,7,22,F,F,,,,,0.00,,F,F,F,Other
+132184,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,752:752,T,F,7,22,F,F,,,,,0.00,,F,F,F,Other
+132185,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,753:753,T,F,7,22,F,F,,,,,0.00,,F,F,F,Other
+132186,3b98604a,GitHub,a0cd6261,data/a0cd6261/other/3b98604a.md,1704:1704,T,F,48,55,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/a0cd6261.csv:360]:
```suggestion
132174,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,193:193,F,F,10,17,F,F,,,Info,,0.00,,F,F,F,Other
132175,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,194:194,F,F,22,29,F,F,,,Info,,0.00,,F,F,F,Other
132176,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,195:195,F,F,9,16,F,F,,,Info,,0.00,,F,F,F,Other
132177,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,249:249,T,F,30,37,F,F,,,Info,,0.00,,F,F,F,Other
132178,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,250:250,T,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
132179,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,742:742,T,F,7,18,F,F,,,Info,,0.00,,F,F,F,Other
132180,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,743:743,T,F,7,18,F,F,,,Info,,0.00,,F,F,F,Other
132181,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,744:744,T,F,7,18,F,F,,,Info,,0.00,,F,F,F,Other
132182,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,745:745,T,F,7,18,F,F,,,Info,,0.00,,F,F,F,Other
132183,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,751:751,T,F,7,22,F,F,,,Info,,0.00,,F,F,F,Other
132184,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,752:752,T,F,7,22,F,F,,,Info,,0.00,,F,F,F,Other
132185,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,753:753,T,F,7,22,F,F,,,Info,,0.00,,F,F,F,Other
132186,3b98604a,GitHub,a0cd6261,data/a0cd6261/other/3b98604a.md,1704:1704,T,F,48,55,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/a0cd6261.csv [hunk 9]
```diff
@@ -340,24 +340,24 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
-132182,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,745:745,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132183,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,751:751,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132184,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,752:752,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132185,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,753:753,T,F,,,F,F,,,,,0.00,,F,F,F,Other
-132186,3b98604a,GitHub,a0cd6261,data/a0cd6261/other/3b98604a.md,1704:1704,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+132174,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,193:193,F,F,10,17,F,F,,,Info,,0.00,,F,F,F,Other
+132175,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,194:194,F,F,22,29,F,F,,,Info,,0.00,,F,F,F,Other
+132176,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,195:195,F,F,9,16,F,F,,,Info,,0.00,,F,F,F,Other
+132177,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,249:249,T,F,30,37,F,F,,,Info,,0.00,,F,F,F,Other
+132178,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,250:250,T,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/a0cd6261.csv:352]:
```suggestion
132178,b0464027,GitHub,a0cd6261,data/a0cd6261/other/b0464027.md,250:250,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
```

**Review by babenek — DISMISSED** [review]
LGTM

**Review by iuriimet — DISMISSED** [review]

File: meta/894e3377.csv [hunk 10]
```diff
@@ -750,26 +750,26 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+132139,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,403:403,F,F,30,39,F,F,,,Info,,0.00,,F,F,F,Other
+132140,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,404:404,F,F,32,41,F,F,,,Info,,0.00,,F,F,F,Other
+132141,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,409:409,F,F,27,36,F,F,,,Info,,0.00,,F,F,F,Other
+132142,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,435:435,F,F,17,28,F,F,,,Info,,0.00,,F,F,F,Other
+132143,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,436:436,F,F,15,26,F,F,,,Info,,0.00,,F,F,F,Other
+132144,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2566:2566,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
+132145,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2567:2567,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
+132146,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2568:2568,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
+132147,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,2569:2569,F,F,5,12,F,F,,,Info,,0.00,,F,F,F,Other
+132148,b679985c,GitHub,894e3377,data/894e3377/test/b679985c.php,236:236,T,F,,,F,F,,,,,0.00,,F,F,F,Other
```
Comment (babenek) [inline meta/894e3377.csv:772]:
```suggestion
132148,b679985c,GitHub,894e3377,data/894e3377/test/b679985c.php,236:236,T,F,92,107,F,F,,,Info,,0.00,,F,F,F,Other
```

**Review by babenek — APPROVED** [review]
LGTM

**Review by iuriimet — APPROVED** [review]

**Pull request merged**